shared/macros-ansible.jinja

```diff
@@ -141,6 +141,26 @@
 {{{ ansible_set_config_file(msg, "/etc/ssh/sshd_config", parameter, value=value, create="yes", prefix_regex='(?i)^\s*', validate="/usr/sbin/sshd -t -f %s", insert_before="^[#\s]*Match") }}}
 {{%- endmacro %}}
 
+{{#
+  High level macro to set a value in a shell-related file that contains var assignments. This
+  takes these values: msg (the name for the Ansible task), path to the file, a parameter to set
+  in the configuration file, and the value to set it to. We specify a case
+  sensitive comparison in the prefix since this is used to deduplicate since
+  We also specify the validation program here; see 'bash -c "help set" | grep -e -n'
+#}}
+{{%- macro ansible_shell_set(path, msg='', parameter='', value='', no_quotes=false) %}}
+{{% if no_quotes -%}}
+{{%- else -%}}
+{{%- set quotes = "\"'" -%}}
+  {{% if "$" in value %}}
+  {{% set value = '"%s"' % value %}}
+  {{% else %}}
+  {{% set value = "'%s'" % value %}}
+  {{% endif %}}
+{{%- endif -%}}
+{{{ ansible_set_config_file(msg, path, parameter, value=value, create="yes", prefix_regex='^\s*', validate="/usr/bin/bash -n %s", insert_before="^[#\s]*Match") }}}
```

Comment:
What is `insert_before="^[#\s]*Match"`? Is it a copy-paste omission?

Reply:
Thanks to pointing this out, I have fixed that after looking at the [Ansible documentation](https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html).